monitor_node_activity.py

```
@@ -0,0 +1,203 @@
+    command = '{} {} -rx "database put {} {} {}"'.format(
+        SUDO, ASTERISK, DB_FAMILY, node_id, sanitized_comment
+    )
+    log_message("Running command: {}".format(command))
+    subprocess.run(command, shell=True, check=True)
+    log_message("Blocked node {} with comment: {}".format(node_id, sanitized_comment))
+
+
+def disconnect_node(node_id, initial_node_id, reason=""):
+    command = '{} {} -rx "rpt fun {} *1{}"'.format(SUDO, ASTERISK, initial_node_id, node_id)
+    log_message("Disconnecting node {} from {}. Reason: {}".format(node_id, initial_node_id, reason))
+    subprocess.run(command, shell=True, check=True)
```

Ruff (0.14.13)

`subprocess` call with `shell=True` identified, security issue

(S602)